File: Back/src/ProEventos.Domain/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? EventDate { get; set; }
        public string Subject { get; set; }
        public int QtyGuests { get; set; }
        
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }

    }
}

File: Front/ProEventos-App/src/app/models/Event.ts
import { Lote } from "./Lote";
import { SocialMedia } from "./SocialMedia";
import { Speaker } from "./Speaker";

export interface Event {
  id: number;
  local: string;
  eventDate?: Date;
  subject: string;
  qtyGuests: number;
  imageURL: string;
  phone: string;
  email: string;
  lotes: Lote[];
  socialMedias: SocialMedia[];
  speakerEvents: Speaker[];
}
